index.js

```diff
@@ -144,22 +145,49 @@ module.exports = function (babel) {
         return newPath.node ? [newPath.get('callee')] : [];
       });
     }
+
+    if (state.opts.ensureModuleApiPolyfill) {
```

Comment:
Can you explain a bit about what you expect this to be used for? When would it be set? when would it not be set? who sets it?

Reply:
This option basically forces the addon to always use the API polyfill. It should be used if the consumer is itself using the API polyfill, and should be disabled otherwise. Since Ember always requires the polyfill, for now it would be set to `true` by `ember-cli-htmlbars`, and as we detangle that we would potentially create a way for `ember-cli-htmlbars` to opt out.

Reply:
kk, gotcha